### ISSUE TITLE
CSV upload

### ISSUE THREAD

**Issue by xinxiang** [campsych/concerto-platform#77]
Followed the tutorial, I created an eleven-question quiz in CSV format for a CAT. When I tried to upload it, there was no error message, but the questions were not there: sometimes there were none, sometimes there were partial, sometimes the sample questions came back. Though after many tries, it finally worked, I don't know why it worked. The same thing happened to another user. There are only two of us trying to make this work. I think it shouldn't be this hard to upload a quiz in CSV format. Where could be the problem?

**Comment by przemyslawlis** [comment]
What Concerto Platform version was it on?
Where did you tried to import it exactly? Data table? CAT wizard item bank? If you're not sure please post a screenshot.
Can you please link your CSV file?

**Comment by xinxiang** [comment]
I am using Concerto v5.0.beta.2.171, I installed this version from the zip file on github that I downloaded on Dec. 3, 2016. The copy I got from git clone command couldn't let me run test.

I followed the CAT Tutorial at https://github.com/campsych/concerto-platform/wiki/CAT-tutorial. When I reached the Item Bank step, I used the Upload CSV button, as shown in the screenshot below.

Here is my CSV file. I renamed it to .txt file so that I can upload it.
[perseid.csv.txt](https://github.com/campsych/concerto-platform/files/650709/perseid.csv.txt)

Here is the screenshot where did the CSV upload:
![screen shot 2016-12-13 at 10 00 08 pm](https://cloud.githubusercontent.com/assets/22461344/21168139/b2bc6b9c-c17f-11e6-9585-ce9025493d69.png)

Thank you for looking into this.
